exercises/concept/high-score/.meta/exemplar.jl

```diff
@@ -0,0 +1,34 @@
+"""
+    add_player!(scores, name, score=0)
+
+Add a player to the score board.
+"""
+add_player!(scores, name, score=0) = push!(scores, name => score)
+
+"""
+    remove_player!(scores, name)
+
+Remove a player from the score board.
+"""
+remove_player!(scores, name) = delete!(scores, name)
+
+"""
+    update_score!(scores, name, score)
+
+Update the score of a player on the score board.
+"""
+update_score!(scores, name, score) = scores[name] = score
+
+"""
+    reset_score!(scores, name)
+
+Reset the score of a player on the score board.
+"""
+reset_score!(scores, name) = update_score!(scores, name, 0)
```

Comment:
Can do three of these at once with this slightly confusing line ;)

```julia
const add_player! = reset_score! = update_score!(scores, name, score=0) = scores[name] = score
```

Reply:
That's pretty neat :D